microprofile-openapi/src/test/java/org/wildfly/quickstarts/microprofile/openapi/BasicRuntimeIT.java

```diff
@@ -26,14 +26,17 @@
 import java.time.Duration;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 /**
- * The very basic runtime integration testing.
+ * The very basic runtime integration test to verify that /openapi context is registered by the OpenAPI subsystem.
+ *
  * @author emartins
+ * @author Radoslav Husar
  */
 public class BasicRuntimeIT {
 
-    private static final String DEFAULT_SERVER_HOST = "http://localhost:8080/microprofile-openapi";
```

Comment:
I will need to double "later" check but I believe this fails the standard distribution testing, where app is not app web root and testing command doesn't override -Dserver.host
that is (for now) the default environment, thus all basic tests have the deployment path in the default const.

Reply:
OK please go ahead and check.

But the `/openapi` is always at the same location...

Reply:
You mean that is something not from the deployment itself? 
Ok I see you change the basic test logic, honestly the idea with those basics were to be the same for all QS and just to check the deployment is running (by checking it's default webpage is there and at most it's content is expected), so maybe keep the old test (yet checking the content) and add this a additional functional one?

Reply:
I will give you a better look and opinion tomorrow, just need to review a pair of PR first in the morning

Reply:
@emmartins Sure, we can do that too. I will close and reopen with a new test.

Reply:
> I will give you a better look and opinion tomorrow, just need to review a pair of PR first in the morning

OK i ll wait till tomorrow, then close and reopen with a new test instead.

Reply:
> I will give you a better look and opinion tomorrow, just need to review a pair of PR first in the morning

@emmartins btw did you make a final decision on this yet?

Reply:
@emmartins No update?